tests/robustness/model/deterministic.go

```diff
@@ -25,7 +25,18 @@ import (
 	"github.com/anishathalye/porcupine"
 )
 
-// DeterministicModel assumes that all requests succeed and have a correct response.
+// DeterministicModel assumes a deterministic execution of etcd requests. All
+// requests that client called where executed and persisted by etcd. This
+// assumption is good for simulating etcd behavior (aka writing a fake), but not
+// for validating correctness as requests might be lost or interrupted. It
+// requires perfect knowledge of what happened to request which is not possible
+// in real systems.
+//
+// Model can still respond with error or incomplete response.
```

Comment:
Nit - I would prefer to align the language around partial / incomplete. Code makes heavy use of word partial so suggest comment stick to that term.

```suggestion
// Model can still respond with error or partial response.
```

Reply:
Done